environments/auth/auth-with-email-lambda-signup-trigger/package.json

```diff
@@ -3,6 +3,6 @@
   "name": "@aws-amplify/ui-auth-with-email-lambda-signup-trigger-environment",
   "version": "0.0.1",
   "scripts": {
-    "pull": "amplify pull --appId d1ytpi9jtd8llt --envName staging"
```

Comment:
I think we'll want to continue testing with Gen1 as well, right? I may be missing something, but it looks like this is removing that capability

Reply:
I combine both to reuse the shell script. It was moved to the end

Reply:
`xxx && echo n | amplify pull --appId d1ytpi9jtd8llt --envName staging"`